packages/ui/.storybook/preview.jsx

```diff
@@ -1,16 +1,12 @@
 import React, { useState } from 'react';
 import globalsCssUrl from '../styles/globals.css?url';
 import penumbraTheme from './penumbraTheme';
-import { ConditionalWrap } from '../src/utils/ConditionalWrap';
+import { ConditionalWrap } from '../src/ConditionalWrap';
 import { PenumbraUIProvider } from '../src/PenumbraUIProvider';
 import { Density } from '../src/Density';
 import { Tabs } from '../src/Tabs';
 import styled from 'styled-components';
 
-const WhiteTextWrapper = styled.div`
```

Comment:
All `<Text />` components now render with `text.primary` color by default, so this is no longer needed.